src/styles/components/_component-card.scss

```diff
@@ -54,18 +56,18 @@
         font-size: map.get($font-sizes, "sm"); // 20px/16
       }
 
-      &::after {
+      &::before {
         content: url("../../../public/images/list_markers/teal-decorative-bullet.svg");
         position: relative;
-        top: 0;
-        left: 1rem; // 16px/16
+        top: -0.1rem;
+        left: -1rem; // 16px/16
       }
 
-      &::before {
+      &::after {
         content: url("../../../public/images/list_markers/teal-decorative-bullet.svg");
         position: relative;
-        top: 0;
-        left: -1rem; // 16px/16
+        top: -0.1rem;
+        left: 1rem; // 16px/16
```

Comment:
These are small adjustments to the 'bullet points' before and after the component headers on the homepage.